components/camel-nats/src/main/java/org/apache/camel/component/nats/NatsConfiguration.java

```diff
@@ -90,6 +90,12 @@ public class NatsConfiguration {
     private boolean traceConnection;
     @UriParam(label = "advanced")
     private HeaderFilterStrategy headerFilterStrategy = new DefaultHeaderFilterStrategy();
+    @UriParam(label = "advanced", defaultValue = "true")
+    private boolean jetstreamEnabled = true;
+    @UriParam(label = "advanced", defaultValue = "default-stream")
+    private String jetstreamName = "default-stream";
```

Comment:
Is this the default name of the stream or something you're assuming?

Reply:
It is not the default name. Rather than leaving it blank, I used a default name.

Reply:
Is it not better to make the end user define the name instead of using `default-stream` which would otherwise be confusing if they dont know why something fails or happens or what stream is actually used by camel with their account.

Reply:
The end user can define the name of the stream. In case, the end user does not define the name of the stream, the default name will be taken. 

Example:
```
.to("nats:" + topic +
                "?jetstreamEnabled=" + streamSettings.isNatsJetstreamEnabled() +
                "&jetstreamName=" + streamSettings.getNatsJetstreamName() +
                "&jetstreamAsync=" + streamSettings.isNatsJetstreamAsyncEnabled() +
                "&secure=true")
        .log("(Device) Message sent to NATS server");
```


What is your suggestion?

Reply:
That they must always define the name to use (no default)